tests/caching/metadata_store/test_memory.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from hamilton.caching.stores.memory import InMemoryMetadataStore
+from hamilton.caching.stores.sqlite import SQLiteMetadataStore
+
+# `metadata_store` is imported but not directly used because it's
+# a pytest fixture automatically provided to tests
+from .test_base import _mock_cache_key, metadata_store  # noqa: F401
+
+
+@pytest.mark.parametrize(
+    "metadata_store",
+    [SQLiteMetadataStore],
```

Comment:
Why parameterize?

Reply:
I was thinking that it would be relevant to test `.persist_to()` and `.load_from()` for all combinations `InMemory x Other`. This could be redundant with unit tests for `Other` implementation and integration testing though

Reply:
OK, given that it's a parameterization of 1, add a comment